app/model/Skautis/Payment/MemberEmailRepository.php

```diff
@@ -27,11 +29,16 @@ public function __construct(Skautis $skautis)
     public function findByMember(int $memberId) : array
     {
         try {
-            $contacts = $this->skautis->org->PersonContactAll(['ID_Person' => $memberId]);
+            $contacts = $this->toArray($this->skautis->org->PersonContactAll(['ID_Person' => $memberId]));
         } catch (PermissionException $e) {
             return [];
         }
 
+        try {
```

Comment:
Existuje důvod, proč mít ty dvě try/catch rozdělené?

Tedy může nastat situace, kdy uživatel má přístup k `PersonContactAll` a nemá přístup k `PersonContactAllParent`?
Pokud ne, tak bych obě ty try/catch spojil

Reply:
myslim ze může a proto jsem to rozděloval

Reply:
A jen takto? Tedy přístup jen k prvnímu nastat nemůže?

Reply:
jen k prvnímu si nemyslim ze to jde. Já to beru ze rodicovske jsou podmnozina osobních kontaktů, tedy kdyz nemas osobní, tak nevracej nic.